src/wasm-lib/kcl/src/parser.rs

```diff
@@ -155,13 +155,28 @@ struct FunctionExpressionResult {
 
 pub struct Parser {
     pub tokens: Vec<Token>,
+    pub unkown_tokens: Vec<Token>,
     stdlib: crate::std::StdLib,
 }
 
 impl Parser {
     pub fn new(tokens: Vec<Token>) -> Self {
         let stdlib = crate::std::StdLib::new();
-        Self { tokens, stdlib }
+        let unkown_tokens: Vec<Token> = tokens
```

Comment:
- 'unkown' should be 'unknown'
- Instead of doing two full iterations, we could use [.partition](https://doc.rust-lang.org/stable/std/iter/trait.Iterator.html#method.partition) method instead to split tokens into two vecs based on whether the tokentype is or is not unknown.